.github/workflows/stale-backtrace-issues.yml

```
@@ -0,0 +1,17 @@
+name: 'Manage stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          days-before-issue-stale: 60
+          days-before-issue-close: 1
+          only-issue-labels: 'backtrace.io'
+          stale-issue-label: 'stale-backtrace'
+          stale-pr-message: 'This issue is stale because it has been open for 60 days with no activity and will be closed in 1 day. To keep it open, leave a comment or remove the "stale" label.'
```

Slightly more concise and straightforward rewording:

```suggestion
          stale-pr-message: 'This issue is stale and will be closed tomorrow if no action is taken. To keep it open, leave a comment or remove the "stale" label.'
```